doc/CHANGELOG.rst

```diff
@@ -174,7 +229,10 @@ v3.2.0
 * Direct Travis to grab the docker image from the cnerg dockerhub account
   instead of Lucas's account. (PR#546)
 
-**Deprecated:** None
+**Deprecated:** 
+* DagMC: Deprecated constructor using a raw pointer for the MBI instance,
```

Comment:
We'll need an empty line between the header and content for this to render correctly I think.

Reply:
done !